robotpy_ext/common_drivers/navx/registerio_i2c.py

```diff
@@ -37,7 +37,10 @@ def init(self):
     
     def write(self, address, value):
         with self.mutex:
-            return self.port.write(address | 0x80, value)
+            aborted = self.port.write(address | 0x80, value)
+        if aborted:
+            logger.warn("navX-MXP I2C write error")
+        return not aborted
```

Comment:
Hm, you're inverting the return value... does nothing check this return value then?

Reply:
Seems that way, oddly enough. I would've expected the return value to be passed up through RegisterIO at least, but apparently not.